FinancialTransactionsApi.Tests/AwsIntegrationTests.cs

```diff
@@ -22,35 +22,8 @@ public class AwsIntegrationTests<TStartup> : IDisposable where TStartup : class
         protected List<Action> CleanupActions { get; set; }
         private readonly List<TableDef> _tables = new List<TableDef>
         {
-            new TableDef()
-            {
-                TableName = "Transactions",
-                PartitionKey = new AttributeDef()
-                {
-                    KeyName = "id",
-                    KeyType = KeyType.HASH,
-                    KeyScalarType = ScalarAttributeType.S
-                },
-                Indices = new List<GlobalIndexDef>()
-                {
-                    new GlobalIndexDef()
-                    {
-                        IndexName = "is_suspense_dx",
-                        KeyName = "is_suspense",
-                        KeyScalarType = ScalarAttributeType.S,
-                        KeyType = KeyType.HASH,
-                        ProjectionType = "ALL"
-                    },
-                    new GlobalIndexDef()
-                    {
-                        IndexName = "target_id_dx",
-                        KeyName = "target_id",
-                        KeyScalarType = ScalarAttributeType.S,
-                        KeyType = KeyType.HASH,
-                        ProjectionType = "ALL"
-                    }
-                }
-            }
+
+            new TableDef { Name = "Transactions", KeyName = "pk", RangeName="id", KeyType = KeyType.HASH,RangeType = KeyType.RANGE,  KeyScalarType= ScalarAttributeType.S}
         };
```

Comment:
According our conversation ...

Reply:
We are creating a report that we know the time frame,I do not think we need ES for this